packages/cra-template-edu/template/src/setupTests.ts

```diff
@@ -1,5 +1,6 @@
 import {setGlobalConfig} from '@storybook/testing-react';
-// @ts-expect-error JS file, which TS is configured to not analyze.
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore JS file, which TS is configured to not analyze.
```

Comment:
Top level tsconfig now sez `allowJs: true`, so this is no longer an error in this repo.

It would still be an error in generated apps, though... so switched to ts-ignore.

Once EDS exports commonjs builds we'll be able to change `allowJs` back to `false` and change this back, too.